value.c

```diff
@@ -240,6 +240,186 @@ bool parse_uservalue_float(const char *nptr, uservalue_t * val)
     return true;
 }
 
+#define DEFAULT_UINTLS_SZ (1000)
+
+/* XXX: this should be in a utility file of some sort (no pun intended) */
+static int _uint64_cmp(const void *i1, const void *i2)
+{
+    return (*(const uint64_t *)i1 - *(const uint64_t *)i2);
+}
+
+bool parse_uservalue_uintset(const char *lptr, uservalue_t *val)
+{
+    const char *tok, *tmpnum, *tmpnum_end;
+    bool        got_num;
+    uint64_t    last_num;
+    uint64_t   *valarr;
+    size_t      arr_szfilled, arr_maxsz;
+    size_t      vaidx = 0;
+    uservalue_t tokval;
+
+    assert(lptr);
+    assert(val);
+
+    enum {
+        NIL = -1,
+        NUM_TOK,
+        RNG_TOK,
+        CMA_TOK,
+    } last_type = NIL;
```

Comment:
Could you be a touch more descriptive? `NUM` is clear, but `RNG` clearly isn't a reference to randomness.

Ok, now I see what they are, the switch below told me. Maybe consider initing them as themselves, like `COMMA_TOK = ','` for clarity.